cscs-checks/mch/automatic_arrays_acc.py

```diff
@@ -84,3 +84,8 @@ def cray_linker_workaround(self):
         if (self.current_system.name == 'dom' and
             self.current_environ.name == 'PrgEnv-cray'):
             self.variables['LINKER_X86_64'] = '/usr/bin/ld'
+
+    @rfm.run_before('compile')
+    def cdt2006_pgi_workaround(self):
+        if self.current_system.name == 'dom':
```

Comment:
I need to further restrict it here to apply the workaround only for PrgEnv-pgi.

Reply:
But this should be harmless to the other PE's no?

Reply:
Yes, in fact, the problem is not due to the PR.